proto/buoyant-cloud-api.proto

```diff
@@ -92,11 +92,69 @@ message Event {
   Workload owner = 3;
 }
 
+//
+// Linkerd Messages
+//
+
+// This message represents PEM encoded certificate
+// data. It might be a single certificate, a pool of
+// roots or a chain of certificates.
+message CertData {
+  bytes raw = 1;
+}
+
+// Represents the certificates that the control plane
+// has been configured with.
+message ControlPlaneCerts {
+  // This is the identity issuer certificate chain
+  CertData issuer_crt_chain = 1;
```

Comment:
take it or leave it: does it make sense to encode each cert in its own cerdata? i.e. so a chain is a `repeated CertData ...`? This is what we do in the proxy-api: https://github.com/linkerd/linkerd2-proxy-api/blob/a7f2efd04dc2905e5591bb68933cb234d39f09c2/proto/identity.proto#L34-L39

Reply:
Makes sense, initially I had it like that because I wanted to avoid parsing the certificate data and juts dump the raw bytes. This has proved to be hard as I get an array of certs from the proxy, and I encode them to a pem blob, ultimately using the utils provided in the linkerd repo. So yes, I might as well have them as separate certs..